BookingProject/BookingWebAPI/DistributedLocking/ServiceExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Redis;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DistributedLocking
+{
+    public static class ServiceExtensions
+    {
+        public static IServiceCollection AddDistributedLocking(this IServiceCollection services)
+        {
+            services.AddScoped<IRedisCacheService, RedisCacheService>();
```

Comment:
TryAdd method should be used bc redis cache service could already be added to app